conf.json

```diff
@@ -1,9 +1,9 @@
-{ settings: 
+{ "settings": 
 	[
 		{
-		  "src_table": "info",
-		  "dest_table": "info",
-		  "column_mapping": {"ID_": "id", "DURATION_": "start_time"},
+		  "src_table": "Camunda_dump_example_csv",
```

Comment:
@Cerfoglg @simonedavico I moved the discussion on the following issue https://github.com/benchflow/benchflow/issues/2. @Cerfoglg please explain there the why of your choice.